### ISSUE TITLE
nri-perfom is bloating event viewer

### ISSUE THREAD

**Issue by veereshdandur** [newrelic/nri-perfmon#49]
nri-perfmon we enabled producing about 200k events in events viewer per day which are mainly informational. It is bloating our events viewer. Is it possible to disable these events?
I am removing the image for security.

**Comment by SpeedAndPower** [comment]
Also came here for same issue. Splunk ingest is through the roof for this info log that should be more of a verbose. There doesn’t seem to be a toggle for it unless I’m missing something

**Comment by PolovicJ** [comment]
Any update to this issue?

**Comment by ricegi** [comment]
Hello there - not at the moment, I'll see what I can do to prioritize this - thx for the nudge

**Comment by jbeveland27** [comment]
Thanks for the patience here. We addressed this with #50 and added a flag for ignoring info logs. Released in [v0.6.2].(https://github.com/newrelic/nri-perfmon/releases/tag/0.6.2):

```
-g | --ignoreInfoLogs [true|false]: Ignores logs of log type Info (default: false)
```

Closing this as completed.
